inputWidget/1.js

```diff
@@ -0,0 +1,22 @@
+const ele = document.querySelector('.inpElement');
+const button = document.querySelector('.numSubmit');
+const numbers = document.querySelector('.numbers');
+
+button.addEventListener('click', function() {
+    let num = Number(ele.value);
+    if(!numbers.innerHTML) {
+        numbers.innerHTML = "";
+    }
+    if(!num) {
+        alert("please enter a valid number and submit");
+    } else if(num > 100 || num < 0) {
+        alert("please enter a number between 0 to 100 and submit")
+    } else {
+        for (; num >=0 ; num--) {
+            let newElement = document.createElement("div");
+            newElement.innerHTML = num;
+            numbers.appendChild(newElement);
```

Comment:
The use case was to show a count down from example 50 and then go down till 0.

Here I am assuming it's appending the number from 50 to 0 in the same span